CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+- Add optional parameter `enableWebAudio: boolean` in the config of meeting manager to decide whether to enable web audio in the device controller.
+- Extend the ability to accept Device and AudioTransformDevice as audio input in meeting manager.
```

Comment:
Could you please backtick Device and AudioTransformDevice here and below in the changed section?

Reply:
Sure, updated